processor/k8sattributesprocessor/doc.go

```diff
@@ -40,7 +40,28 @@
 //
 // If Pod association rules are not configured resources are associated with metadata only by connection's IP Address.
 //
-//
+// Which metadata to collect is determined by `metadata` configuration that defines list of resource attributes
+// to be added. Items in the list called exactly the same as the resource attributes that will be added.
+// The following list of attributes is enabled by default if `metadata` configuration is not specified:
+//   - k8s.namespace.name
+//   - k8s.pod.name
+//   - k8s.pod.uid
+//   - k8s.pod.start_time
+//   - k8s.deployment.name
+//   - k8s.cluster.name
+//   - k8s.node.name
+// Not all the attributes are guaranteed to be added. For example `k8s.cluster.name` usually is not provided by k8s API,
+// so likely it won't be set as an attribute.
+
+// The following attributes are not included by default, but can be enabled with `metadata` config option.
```

Comment:
Any reason to not enable container metadata by default as well? In most cases when `container.name` is not present this would be a noop but would work when the identifying attribute is present. If it doesn't incur an additional significant perf penalty, I think we should do it.

Reply:
Sounds good. I'll add them by default